libmscore/cmd.cpp

```diff
@@ -2301,6 +2303,11 @@ Element* Score::move(const QString& cmd)
                                     }
                         break;
                         }
+                  case ElementType::HBOX: Q_FALLTHROUGH();
```

Comment:
If we're trying to get rid of Qt in libmscore, maybe just use a comment?
                      case ElementType::HBOX: // fallthrough

Reply:
Are we?

Reply:
that's a question for @igorkorsukov

Reply:
Right, we want to reduce our dependency on Qt where possible.
But we want to do this evolutionarily, so now the rule is this: where Qt is already used, you can use Qt, where it does not use, you do not need to use Qt. 
So far, everything is simple.

Reply:
Ok, I changed these to` // fallthrough`
There's really only about 3 different Q_FALLTHROUGH(); statements in libmscore/ directory. I'm almost tempted to just convert them here to be done with it, but I'll leave it alone.